modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -3475,10 +3475,10 @@ export const console: ConsoleNS = {
                                 }
                             },
                             useNumericChars: {
```

Comment:
Shall we rename this as well?

Reply:
Resolve with 41555a63bde603bb36b8c7b0c1acf48eea654946